src/amo/components/Collection/index.js

```diff
@@ -263,6 +287,23 @@ export class CollectionBase extends React.Component<Props> {
     /* eslint-enable react/no-danger */
   }
 
+  renderDeleteButton() {
+    const { hasEditPermission, i18n } = this.props;
+    if (!hasEditPermission) {
+      return null;
+    }
+    return (
```

Comment:
nit: maybe a blank line before the `return` statement would be nice for readability.